install.sh

```diff
@@ -4,33 +4,50 @@
 # Remove ads system-wide in Linux                         #
 ###########################################################
 # authors:      sedrubal, diy-electronics                 #
-# version:      v3.1                                      #
+# version:      v3.2                                      #
 # licence:      CC BY-SA 4.0                              #
 # github:       https://github.com/sedrubal/adaway-linux  #
 ###########################################################
 
 # settings
 HOSTS_ORIG="/etc/.hosts.original"
+SYSTEMD_DIR="/etc/systemd/system"
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )" # Gets the location of the script
 SRCLST="$DIR/hostssources.lst"
-VERSION="3.1"
+VERSION="3.2"
 #
 
+
 set -e
 
+# check root
+if [ "$(id -u)" != "0" ] ; then
```

Comment:
You can also just use `${UID}`.

Reply:
I'm not yet convinced if we should only allow to run this as root or if we just want to use `sudo` for every command which needs root permissions... One side effect of this is:

```sh
$ ./install.sh --help
This script must be run as root
```